soc/nordic/nrf54h/Kconfig

```diff
@@ -63,6 +63,13 @@ config SOC_NRF54H20_CPURAD_COMMON
 	select HAS_PM
 	select HAS_POWEROFF
 
+config SOC_NRF54H20_CPURAD_ENABLE
+	bool "NRF54H20 Radio core is enabled at boot time"
```

Comment:
nit: "Enable nRF54H20 Radio core at boot time"

Reply:
This broke CI.

Some kind of rule about prompts starting with Enable.